### ISSUE TITLE
kong don't start

### ISSUE THREAD

**Issue by median-dxz** [kubernetes/dashboard#8909]
### What happened?

Firstly, I know that it's a duplicate of #8765 and #8846.
But I'm using Kubernetes on Docker Desktop for Windows and I faced exactly the same problem. And I have tried clearing the previous minikube installnation thoroughly but still have the bind 8444 error.
It might not be minikube causing the problem？

### What did you expect to happen?

Kong started normally

### How can we reproduce it (as minimally and precisely as possible)?

exec helm command from readme

### Kubernetes Dashboard version

7.3.1

### Kubernetes version

Client Version: v1.29.1 Kustomize Version: v5.0.4-0.20230601165947-6ce0bf390ce3 Server Version: v1.29.1

**Comment by floreks** [comment]
I can't tell you what is occupying this port for you. Only you can debug your system and figure it out. You either free up this port or override the port that kong is using by default via helm chart values.

There is nothing we can do here since this is a configuration issue on your side.

**Comment by median-dxz** [comment]
> I can't tell you what is occupying this port for you. Only you can debug your system and figure it out. You either free up this port or override the port that kong is using by default via helm chart values.
> 
> There is nothing we can do here since this is a configuration issue on your side.

Thanks you patiently reply.
I'm still learning about k8s, if you can provide some info so that I can troubleshoot the truely problem, I'll be very appreciately. I just have no idea how check out "port occupying" in a cluster.
Of course, there might not a suitable place for such a discuss, you can close this issue if needed.

**Comment by AndeYashwanth** [comment]
I faced this issue in docker desktop windows 4.28.0 yeaterday when using the helm method. So I used the older 2.7.x version with yaml file instead and it was working fine.
I tried clean installing docker desktop and also updated to 4.29.0 in an attempt to fix some other unrelated issue. Then I tried installing latest kubernetes dashboard 7.x and it was working fine.

**Comment by median-dxz** [comment]
Hi, @AndeYashwanth 
That also works for me! Docker Desktop just has released 4.29.0 recently (it's a little weird that the docker official release notes page shows this version was released in 04-08 but I can't receive it when I opened this issue). And after update from 4.28.0 to 4.29.0, I could start Kong with Helm and default values (no need to reinstall Docker Desktop entirely). It's proved that it was Docker Desktop causing this issue.
Thank you for your help!
